### PR TITLE
make code of pages better

### PR DESCRIPTION
**Pull request by akisblack**

### CODE REVIEW

**Comment by Odyssey346** [comment]
istfg i changed the background, oh well I can do that later. Let me take a look

**Comment by Odyssey346** [comment]
Thanks for using <pre> tags, but I don't think the rest is necessary. Please revert those changes (except for the pre tag stuff ofc)

**Comment by akisblack** [comment]
stop using formal language

File: src/routes/brave.svelte [hunk 1]
```diff
@@ -1,13 +1,20 @@
+<script>
+    let browserName = "BRAVE BROWSER";
+    let browserNameHTML = `<b name="insert">${browserName}</b>`;
+</script>
```
Comment (Odyssey346) [inline src/routes/brave.svelte:4]: Not necessary

File: src/routes/index.svelte [hunk 2]
```diff
@@ -1,13 +1,20 @@
+<script>
+    let browserName = "VIVALDI BROWSER";
+    let browserNameHTML = `<b name="insert">${browserName}</b>`;
+</script>
+
```
Comment (Odyssey346) [inline src/routes/index.svelte:5]: Not necessary

**Comment by akisblack** [comment]
also, only reason i did the other shit was for ease of use if a person wants to contribute

**Review by Odyssey346 — CHANGES_REQUESTED** [review]
b

**Comment by Odyssey346** [comment]
> also, only reason i did the other shit was for ease of use if a person wants to contribute

it doesnt help much

File: .gitignore [hunk 3]
```diff
@@ -2,4 +2,5 @@
 node_modules
 /.svelte-kit
 /package
-build
\ No newline at end of file
+build
+shit.svelte
```
Comment (Odyssey346) [inline .gitignore:6]: Can you explain what this is?

**Review by Odyssey346 — CHANGES_REQUESTED** [review]
ok

**Comment by Odyssey346** [comment]
I've tested this change locally, it's completely broken.
image

**Comment by Odyssey346** [comment]
Revert that commit - I've already fixed it on my end.

**Comment by Odyssey346** [comment]
BrowserCopypasta to the moon :rocket:

**Pull request merged**